runtime/include/chpl-tasks.h

```diff
@@ -303,6 +303,135 @@ size_t chpl_task_getDefaultCallStackSize(void);
 extern void chpl_taskRunningCntInc(int64_t _ln, c_string _fn);
 extern void chpl_taskRunningCntDec(int64_t _ln, c_string _fn);
 
+//
+// Tasking callback support.
+//
+// NAME
+//
+//   chpl_task_install_callback   - install tasking callback function
+//   chpl_task_uninstall_callback - remove tasking callback function
+//
+//
+// SYNOPSIS
+//
+//     #include "chpl-tasks.h"
+//
+//     int chpl_task_install_callback(chpl_task_cb_kind_t kind,
+//                                    chpl_task_cb_fn_t cb_fn);
+//     int chpl_task_uninstall_callback(chpl_task_cb_kind_t kind,
+//                                      chpl_task_cb_fn_t cb_fn);
+//
+//
+// DESCRIPTION
+//
+//   These runtime functions are used to install and remove functions
+//   called when certain events occur in the runtime tasking layer.
+//
+//   The 'kind' argument specifies the tasking layer event that is of
+//   interest.  The 'cb_fn' argument is a pointer to a callback function
+//   for that event.  The callback functions installed for each event
+//   will be called, in the order in which they were installed, each
+//   time the corresponding event occurs in the tasking layer of the
+//   runtime.
```

Comment:
Good point. These must be called per top-level locale (tasking layer instance, basically), since the tasking layer is an intra-locale thing. I'll update the comment to say this.